### PR TITLE
Remove countries

### PR DESCRIPTION
**Pull request by lasong**

### CODE REVIEW

File: src/client/components/SupplierApplicationForm/SupplierApplicationForm.js [hunk 1]
```diff
@@ -30,31 +29,7 @@ class SupplierApplicationForm extends React.Component {
...
-    countriesPromise.then(countries => {
-      this.setState({
-        isLoading: false,
-        countries
-      })
-    });
-
-    console.log('this.state', this.state);
-  }
+  state = { key: 1 }
```
Comment (patrykkopycinski) [inline src/client/components/SupplierApplicationForm/SupplierApplicationForm.js]: do we need that?

File: src/client/components/SupplierRegistrationForm/SupplierRegistrationForm.js [hunk 2]
```diff
@@ -55,16 +45,14 @@ class SupplierRegistrationForm extends React.Component {
...
-          onboardData
-        })
-      });
+      this.setState({
+        isLoading: false,
+        onboardData
+      })
+    });
 
     console.log('this.state', this.state);
```
Comment (patrykkopycinski) [inline src/client/components/SupplierRegistrationForm/SupplierRegistrationForm.js]: please remove this line

File: src/client/service/ApplicationFormService.js [hunk 3]
```diff
@@ -4,14 +4,4 @@ export default class ApplicationFormService {
   constructor(actionUrl) {
```
Comment (patrykkopycinski) [inline src/client/service/ApplicationFormService.js]: i think we don't need this file at all

Comment (lasong) [inline src/client/service/ApplicationFormService.js]: 👍

File: src/client/components/SupplierApplicationForm/SupplierApplicationForm.js (hunk 1 above)
Comment (lasong) [inline src/client/components/SupplierApplicationForm/SupplierApplicationForm.js]: Yes we do. It's used for react-bootstrap tabs https://react-bootstrap.github.io/components.html#tabs-controlled

Comment (patrykkopycinski) [inline src/client/components/SupplierApplicationForm/SupplierApplicationForm.js]: please rename it to something more meaningful then

**Comment by patrykkopycinski** [comment]
@lasong remember to remove branch after the PR is merged

**Pull request merged**